wrappers/android/tools/camera/src/main/java/com/intel/realsense/camera/FirmwareUpdateDialog.java

```diff
@@ -93,6 +98,9 @@ public void onClick(View view) {
             }
         });
 
+        if (mAlreadyLatestInstalled)
```

Comment:
In multicam scenario this approach can't work.
Is this anticipated?

Reply:
The whole camera application assumes that only one device is connected. We have the multicam application for android example of multicam application